akka/src/main/scala/anorm/AkkaStream.scala

```diff
@@ -50,7 +56,7 @@ object AkkaStream {
    * @param m $materializerParam
    * @param connection $connectionParam
```

Comment:
Yes, and would be nice to use a Scaladoc `@define` to make sure this same explanation is used for all the functions